src/gopher.cc

```diff
@@ -102,6 +102,9 @@ class GopherStateData
 
     ~GopherStateData();
 
+    /// URL for icon to display, given the item-type code
+    const char *iconUrl(const char);
```

Comment:
This particular `const` is not a part of the API. It is a part of that API _implementation_. IIRC, some compilers will even complain about such extraneous `const`s.

```suggestion
    const char *iconUrl(char);
```